src/web/routes/application/check/get.test.js

```diff
@@ -1,5 +1,41 @@
 const test = require('tape')
-const { getRowData, getLastStepPath, getFlattenedRowData } = require('./get')
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+
+const { getRowData, getFlattenedRowData } = require('./get')
+
+const getPreviousPath = sinon.spy()
+const { getLastNavigablePath } = proxyquire('./get', { '../common/get-previous-path': { getPreviousPath } })
+
+test('getLastNavigablePath returns path of last step if last step has not isNavigable function', (t) => {
+  const steps = [{ path: '/first' }, { path: '/last' }]
+  const session = {}
+
+  const result = getLastNavigablePath(steps, session)
+
+  t.equal(result, '/last')
+  t.end()
+})
+
+test('getLastNavigablePath returns path of last step if last step isNavigable return true', (t) => {
```

Comment:
This doesn't quite make sense, I think should be something like `returns path of last step if last step's isNavigable function returns true`

Reply:
Updated.